main.nf

```diff
@@ -81,83 +123,360 @@ if( !(workflow.runName ==~ /[a-z]+_[a-z]+/) ){
   custom_runName = workflow.runName
 }
 
+/*
+ * short read preprocessing options
+ */
+params.adapter_forward = "AGATCGGAAGAGCACACGTCTGAACTCCAGTCA"
+params.adapter_reverse = "AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGT"
+params.mean_quality = 15
+params.trimming_quality = 15
+params.keep_phix = false
+// params.phix_reference = "ftp://ftp.ncbi.nlm.nih.gov/genomes/genbank/viral/Enterobacteria_phage_phiX174_sensu_lato/all_assembly_versions/GCA_002596845.1_ASM259684v1/GCA_002596845.1_ASM259684v1_genomic.fna.gz"
+params.phix_reference = "$baseDir/assets/data/GCA_002596845.1_ASM259684v1_genomic.fna.gz"
+
+/*
+ * binning options
+ */
+params.skip_binning = false
```

Comment:
All of these defaults should go in `nextflow.config` instead of `main.nf`

The only exceptions are where we are setting `params.something` with more complex logic. For example, the iGenomes references.